Tracker/Targets.php

```diff
@@ -36,6 +36,7 @@ public function getTargets()
             $targets[] = array(
                 'url' => 'http://demo-anonymous.piwik.org/piwik.php',
                 'idSite' => 1,
+                'anon' => true
```

Comment:
Can we use the full word `useAnonymization` or so? It's otherwise hard to know what `anon` means